app/src/main/kotlin/social/plasma/models/Contact.kt

```diff
@@ -0,0 +1,32 @@
+package social.plasma.models
+
+import okio.ByteString
+import okio.ByteString.Companion.decodeHex
+import java.lang.IllegalArgumentException
+
+data class Contact(
+    val pubKey: ByteString,
+    val homeRelayUrl: String? = null,
+    val petName: String? = null,
+) {
+    companion object {
+        private fun String.pubKeyOrNull(): ByteString? =
+            try { this.decodeHex() }
+            catch (_: IllegalArgumentException) { null }
+
+        fun fromTag(tag: List<String>): Contact? =
```

Comment:
I'm not sure why, but contact list (nip-02) is encoded in the tags, rather than a JSON doc in the content. That makes it cumbersome to parse.